isonet/protocols/protocol_tomo_reconstruction.py

```diff
@@ -272,7 +272,7 @@ def prepareProjectStep(self):
             tomoName = tomo.getTsId()
             tomoLnName = os.path.join(self.tomoPath, tomoName + '.mrc')
             if not os.path.exists(tomoLnName):
-                os.link(tomofn, tomoLnName)
+                os.symlink(tomofn, tomoLnName)
```

Comment:
Hi, @rdrighetto,
I agree. But I suggest using the method 'createLink' from pyworkflow > utils > path.py, as it is more robust and covers cases such as the step fails and the user wants to continue, but the link already exists.

Reply:
Done!